### PR TITLE
Use styled mxids in member list

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/14825

Screenshot_20210707_182644

Preview: https://pr6328--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by ShadowJonathan** [comment]
Wouldn't it style better if the disambiguated username goes *below* the name displayed there?

Else it'll be cut off if the displayname is too long.

(For that matter, wouldn't it make sense to *always* show usernames under displaynames in the sidebar? I think it warrants to always display info like that in a sidebar)

**Comment by t3chguy** [comment]
Keep in mind below is already uses for status/presence

image

**Comment by t3chguy** [comment]
> For that matter, wouldn't it make sense to always show usernames under displaynames in the sidebar?

No, hiding MXIDs in as many places as is sane is very much intentional.

**Comment by ShadowJonathan** [comment]
Good point, though i still think it cutting off if the displayname is too long could be an annoyance.

**Comment by aaronraimist** [comment]
They already get cut off in the current version. At least the font size is a bit smaller here so it will require more characters to be cut off. If the user doesn't want it to cut off the usernames they can expand the right panel.

File: res/css/_components.scss
Comment (turt2live) [inline res/css/_components.scss:183]: please run `rethemedex.sh` so these get ordered correctly

**Review by turt2live — APPROVED** [review]
code generally seems fine

File: res/css/_components.scss
Comment (SimonBrandner) [reply res/css/_components.scss:183]: I actually had no idea this was a thing until recently :(

**Comment by ara4n** [comment]
This has been blocked on review by design for 9 months (cc @nadonomy), and it's a concrete useful security feature (unspoofable disambiguation) which from the screenshots is a clear cosmetic improvement.  So I'm making an exec decision to approve it. @SimonBrandner - thanks for doing it, and sorry it got stuck for so long.

**Review by ara4n — APPROVED** [review]

**Comment by ara4n** [comment]
(have tried to fix the merge commit from within GH issue UI 'blind', hopefully it worked... :S)

**Comment by turt2live** [comment]
Looks like it's failing modern code checks - will take a look shortly.

**Comment by turt2live** [comment]
ftr, this missed https://github.com/matrix-org/matrix-react-sdk/pull/7595 but have re-applied it. From looking at the merge diff, it's the only feature at risk.

*Should* be safe to include this in the RC tomorrow. If not, it's an easy revert and something we can re-apply onto develop.

**Comment by turt2live** [comment]
this ended up getting reverted - see https://github.com/matrix-org/matrix-react-sdk/pull/8107 for details

**Pull request merged**
